tools/integration_tests/util/setup/implicit_and_explicit_dir_setup/implicit_and_explicit_dir_setup.go

```diff
@@ -72,15 +77,55 @@ func RemoveAndCheckIfDirIsDeleted(dirPath string, dirName string, t *testing.T)
 	}
 }
 
-func CreateImplicitDirectoryStructure(testDir string) {
+type objectCreationMetadata struct{ content, completeObjectName string }
+
+func createObjectsOnGcs(objects []objectCreationMetadata, t *testing.T, sgClient *storage.Client) {
```

Comment:
Also the order should be 
```
Objects, client, t
```
to maintain consistency with other tests

Reply:
done

Reply:
done